package.json

```diff
@@ -4,17 +4,19 @@
     "start:ui": "static static",
     "start:sign": "tsx ./generate-permit/generate-permit2-url.ts",
     "utils:hash": "git rev-parse HEAD > static/commit.txt",
-    "build": "run-s utils:hash",
+    "build": "yarn utils:hash && tsx build.ts",
```

Comment:
This seems unexpected. You can do either:

1.

`"prebuild": "utils:hash"`
`"build": "tsx build.ts"`

---

2.

`"utils:hash": "git rev-parse HEAD > static/commit.txt"`
`"utils:build": "tsx build"`
`"build": "run-s utils:hash utils:build"`

Reply:
the prebuild script didn't work for me for some reason @pavlovcik

Reply:
That is built in functionality for npm. I'm pretty sure it works with yarn automatically. What about just using `run-s`?